### PR TITLE
Fixup version bumper job

### PR DESCRIPTION
**Pull request by SuperQ**
Move role repo discovery to a script to avoid shell escaping issues.

### CODE REVIEW

File: discover_role_repos.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+for defaults_file in roles/*/defaults/main.yml ; do
+  role="$(echo "${defaults_file}" | cut -f2 -d'/')"
+  yq eval "[{\"repo\": .${role}_repo, \"role\": \"${role}\"}]" "${defaults_file}"
+done | yq -o json -I=0
```
Comment (gardar) [inline discover_role_repos.sh:6]: Shouldn't this be `vars/main.yml` now that the repo var lives there?

Comment (SuperQ) [reply discover_role_repos.sh:6]: Yes, forgot to update that.

**Review by gardar — COMMENTED** [review]
I think we should avoid polluting the root of the repo with ci related scripts and I suggest moving `discover_role_repos.sh -> .github/scripts/discover_role_repos.sh`
Which is the suggested script location from [github](https://docs.github.com/en/actions/learn-github-actions/essential-features-of-github-actions#adding-scripts-to-your-workflow)

(And we should move `version_updater.sh` too)

**Comment by SuperQ** [comment]
Done, moved scripts.

**Review by gardar — APPROVED** [review]

**Pull request merged**